module_utils/network_lsr/ethtool.py

```diff
@@ -46,6 +50,8 @@ def get_perm_addr(ifname):
             res = ecmd.tobytes()
         except AttributeError:  # tobytes() is not available in python2
             res = ecmd.tostring()
+        # disable check for bad names:
+        # pylint: disable=C0102
         _, size, perm_addr = struct.unpack("II%is" % MAX_ADDR_LEN, res)
```

Comment:
Is `_` now considered to be bad? Then this would allow to avoid this AFAICS:

```suggestion
        size, perm_addr = struct.unpack("II%is" % MAX_ADDR_LEN, res)[1:]
```

but I am not 100% sure that this is better to read.

Reply:
> Is `_` now considered to be bad?

Apparently so?  Which I don't understand - it is the standard, or at least de facto, way to ignore a return value from a function.  And there seems to be no way to tell pylint to accept it.

> Then this would allow to avoid this AFAICS:
> 
> but I am not 100% sure that this is better to read.

Another alternative
```python
size, perm_addr = struct.unpack("I%is" % MAX_ADDR_LEN, res[4:])
```

Reply:
There doesn't seem to be a format specifier or modifier that I can use to say "match an `I` here but don't return the value" - https://docs.python.org/2/library/struct.html

Reply:
@tyll so what do you want to do?
* keep the `_` with the pylint exception
* use `struct.unpack()[1:]
* use `struct.unpack(res[4:])`
* something else?

Reply:
I would like to know why `_` is considered bad. It is not in the list of bad names in https://github.com/linux-system-roles/network/blob/main/pylintrc#L353 and when I run pylint locally, the C0102 warning does not appear - it does when I switch `_` to `foo`. Do you have any idea where it comes from? Does tox-lsr override this somehow?

Reply:
> Does tox-lsr override this somehow?

I think it must be `ansible-test` - I'm not sure why ansible would care that `_` is being used in a module.  I'll have to dig into the ansible-test source code to figure out how it is invoking pylint.

Reply:
https://github.com/ansible/ansible/blob/stable-2.9/test/lib/ansible_test/_data/sanity/pylint/config/ansible-test.cfg#L28
I've asked the ansible team why - waiting on a reply

Reply:
In their custom check script they wrote to use `dummy` instead of `_`: [PR](https://github.com/ansible/ansible/pull/54140/commits/7928dba817271802077b95881b33209bfb2ffb69#diff-b91efbf85963e341f0494813aa21ca602b60097cf080310de1f707acebfef2bb), see test/sanity/code-smell/no-underscore-variable.py

Reply:
Ansible devs say "its also 'standard for gettext'" https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/no-underscore-variable.html - Ansible reserves the right to use `_` for its own future use

Ansible devs suggest to use `dummy` - so
```
+        dummy, size, perm_addr = struct.unpack("II%is" % MAX_ADDR_LEN, res)
```
this passes with `tox -e pylint`
I also checked if using the word "dummy" for a variable name is problematic with any conscious language initiatives - I haven't found any

@tyll ok to use `dummy` here?

Reply:
@richm dummy is fine by me, thanks

Reply:
fixed in https://github.com/linux-system-roles/network/pull/362/commits/e70dd1b04722fc2f617bdb2745f54b6312f56c18